src/components/DailyStatistics/TransactionHistoryCard.tsx

```diff
@@ -58,6 +59,8 @@ export const TransactionHistoryCardComponent: FunctionComponent<TransactionHisto
   transactionHistory,
   loading,
 }) => {
+  const { i18nt } = useTranslate();
+
   return (
     <Card style={styles.stats}>
```

Comment:
noticed the name of the item was not translated here: 

![image](https://user-images.githubusercontent.com/43963814/121140383-5bf8fe80-c86c-11eb-957d-3998ec8c09f3.png)

Reply:
fixed! Tested it out on food-stg